src/core/graph.js

```diff
@@ -575,6 +763,18 @@ export class Graph {
     return result;
   }
 
+  /**
+   * Allows merging multiple graphs.
```

Comment:
```
   * Allows merging multiple graphs.
```

It doesn’t “allow” anything; it actually merges the graphs! And I can
infer that from the method name—if I’m reading these docs, I’m probably
trying to figure out what “merging” graphs actually means. This is not a
helpful summary.

Suggest:

> Compute the union of the given graphs. The result is a new graph that
> has all the nodes and edges from all the provided graphs.
>
> If two of the given graphs have edges with the same address, the edges
> must be equal (i.e., must have the same source and destination in each
> graph). If this is not the case, an error will be thrown.

```
   *
   * Example usage:
   * const g1 = new Graph().addNode(a).addNode(b).addEdge(x);
   * const g2 = new Graph().addNode(c);
   * const g3 = Graph.merge([g1, g2]);
```

Good to have an example, but this example isn’t particularly meaningful. In particular, the two graphs that are merged are disjoint to begin with. Suggest something like:

> ```js
> const g1 = new Graph().addNode(a).addNode(b).addEdge(e);
> const g2 = new Graph().addNode(b).addNode(c).addEdge(f);
> const g3 = Graph.merge([g1, g2]);
> Array.from(g3.nodes()).length;  // 3
> Array.from(g3.edges()).length;  // 2
> ```

```
   *
   * The new Graph will contain the union of all the nodes and edges in each of
   * the merged graphs. It is a separate instance and may be mutated
   * independently.
```

Good (though may merit some rewording in response to comments on first
paragraph).